app/assets/javascripts/forever_style_guide/account-popover.js

```diff
@@ -0,0 +1,42 @@
+// Trigger account popovers
+$(function () {
+  $('[data-toggle="popover-desktop"]').popover({
+    html: true,
+    content: function () {
+      return $('.account-popover').html();
+    },
+    template: '<div class="popover account-popover hidden-sm hidden-xs"><div class="arrow"></div><div class="popover-content"></div></div>',
```

Comment:
I'm highly tempted to think about this as a .js.erb file instead so that the template could be included via a partial.  However given the way this will integrate with the store I think that would add an extra level of complexity there...